src/palace/manager/sqlalchemy/model/licensing.py

```diff
@@ -205,6 +205,38 @@ def checkin(self) -> None:
             logging.warning(f"Checking in expired license # {self.identifier}.")
 
 
+class LicensePoolType(StrEnum):
+    """The licensing model for a licensepool."""
+
+    METERED = auto()
+    # Library has finite copies/checkouts available
+
+    UNLIMITED = auto()
+    # Unlimited access while the license is active
```

Comment:
Note to self: Does this include aggregated books that have at least one license with an expiration date, but no checkout/concurrency limits?

If so, it would be nice to be as clear as possible about what each status means.